pkg/sql/pg_catalog_diff.go

```diff
@@ -185,6 +188,30 @@ func (p PGCatalogTables) rewriteDiffs(diffFile string) error {
 	return nil
 }
 
+// getNotImplementedTables retrieves tables that are not yet part of crdb
+func (p PGCatalogTables) getNotImplementedTables(source PGCatalogTables) PGCatalogTables {
```

Comment:
Can you add a note stating where source is coming from so in the future we can easily tell what's being compared (crdb current tables, and pg as source)

Reply:
Done.